lib/fluent/plugin/out_uri_decode.rb

```diff
@@ -1,16 +1,16 @@
-class Fluent::URIDecoder < Fluent::Output
+require 'fluent/plugin/output'
+require 'uri'
+
+class Fluent::Plugin::URIDecoderOutput < Fluent::Plugin::Output
   Fluent::Plugin.register_output('uri_decode', self)
 
-  config_param :tag, :string, :default => nil
-  config_param :remove_prefix, :string, :default => nil
-  config_param :add_prefix, :string, :default => nil
-  config_param :key_name, :string, :default => nil
-  config_param :key_names, :string, :default => ''
+  helpers :event_emitter
 
-  def initialize
-    super
-    require 'uri'
-  end
+  config_param :tag, :string, default: nil
+  config_param :remove_prefix, :string, default: nil
```

Comment:
Why not display warning with `deprecated: "use @label instead for event routing"` like Fluentd?

ref: https://github.com/fluent/fluentd/blob/e3edf3e3754638be8cb513cebacf2d1fa5042d8c/lib/fluent/plugin/out_exec_filter.rb#L31